gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.warning.mode=all
 
-currentVersion=2.5.1
+currentVersion=2.5.1-SNAPSHOT
```

Comment:
should be `2.5.2-SNAPSHOT`, since version `2.5.1` just has been released and the next release likely will be called `2.5.2`